code/modules/mob/living/carbon/carbon.dm

```diff
@@ -7,13 +7,17 @@
 	update_body_parts() //to update the carbon's new bodyparts appearance
 
 /mob/living/carbon/Destroy()
+	//This must be done first, so the mob ghosts correctly before DNA etc is nulled
+	. =  ..()
+
 	QDEL_LIST(internal_organs)
 	QDEL_LIST(stomach_contents)
 	QDEL_LIST(bodyparts)
 	QDEL_LIST(implants)
 	remove_from_all_data_huds()
 	QDEL_NULL(dna)
-	return ..()
+
+	return .

```

Comment:
???????????????????????????????????????????????????????

Reply:
explicit is better than implicit

Reply:
better how? I think this is ugly af but maybe you mean something else

Reply:
it's better because it's obvious at first glance to everyone

Reply:
if a person doesn't know what `. = ..()` is, then they really shouldn't be touching these things